packages/compartment-mapper/src/import-hook.js

```diff
@@ -214,7 +220,7 @@ export const makeImportHookMaker = (
             sourceLocation: moduleLocation,
             parser,
             bytes: transformedBytes,
-            record,
+            record: concreteRecord,
```

Comment:
this is for bundling

Reply:
I’m convinced this is the right way to go about this for now. Bundling will eventually need to be able to follow aliases through the module descriptor graph, either by shortening paths in pre-processing or threading descriptors into the bundle. Probably the former.